### PR TITLE
Update Docs (rules, and collections)

### PR DESCRIPTION
**Pull request by ifestus**
**Summary:** Summary of changes

## Changes

* update references to `collection`, `provider`, and `rule` to reflect Cumulus 1.9.0 release.

Reviewers: @tag-your-friends

### CODE REVIEW

File: docs/data-cookbooks/setup.md [hunk 1]
```diff
@@ -71,23 +72,36 @@ _The above optional attributes are not shown in the example provided, but they h
...
-|Rule state|`ENABLED`|Yes|<code>(ENABLED&#124;DISABLED)</code> whether or not the rule will be active|
-|Optional tags|`"nightly"`|No|A string type tag that can be added to simplify search|
-
-#### rule-value
+|name|`"L2_HR_PIXC_kinesisRule"`|Yes|Name of the rule. This is the name under which the rule will be listed on the dashboard|
+|workflow|`"KinesisTriggerTest"`|Yes|Name of the workflow to be run. A list of available workflows can be found on the Workflows page|
+|provider|`"PODAAC_SWOT"`|Yes|Configured provider's ID. This can be found on the Providerse page|
+|collection|`<JSON Object>` collection object shown [below](#collection-object)|Yes|Name and version of the collection this rule will moderate. Configured and found in the Collections page|
+|rule|`<JSON Object>` rule type and associated values - discussed [below](#rule-object)
+|state|`ENABLED`|Yes|<code>(ENABLED&#124;DISABLED)</code> whether or not the rule will be active|
```
[the remark this answers is not in the document]
Comment (flamingbear) [reply docs/data-cookbooks/setup.md]: @laurenfrederick 
Does that mean that "state" should also be pulled out of the rule example file. `example/data/rules/L2_HR_PIXC_kinesisRule.json`? ......

Actually I just tested this, and and doesn't enable the mapping.  Delete the state from the rule, and run `CleansUpTestResources.js` & `PopulateProvidersCollections.js` , you'll see on the lambda console that the kinesis stream is linked, but not enabled.  same with the rule in dynamoDB.

```{
  "collection": {
    "name": "L2_HR_PIXC",
    "version": "000"
  },
  "createdAt": 1533917413696,
  "name": "L2_HR_PIXC_kinesisRule",
  "provider": "PODAAC_SWOT",
  "rule": {
    "arn": "b155c832-7004-4c1f-84bf-e1450a32ee1a",
    "type": "kinesis",
    "value": "arn:aws:kinesis:us-east-1:893015583569:stream/mhs-cumulus-testStream"
  },
  "updatedAt": 1533917413696,
  "workflow": "KinesisTriggerTest"
}
```

This sounds like you think it's a bug, and should be fixed.  I think that makes sense.

[the remark this answers is not in the document]
Comment (flamingbear) [reply docs/data-cookbooks/setup.md]: Yeah, in addKinesisEvent source the mapping is created with a test for enabled here
https://github.com/nasa/cumulus/blob/master/packages/api/models/rules.js#L186

Would be an easy fix to force a new mapping to be enabled by default, but overridable I assume.

File: docs/data-cookbooks/setup.md [hunk 2]
```diff
@@ -71,23 +72,36 @@ _The above optional attributes are not shown in the example provided, but they h
...
+  "version": "000"
+}
+```
+
+#### rule-object
+|Key|Value|Required|Description|
+|---|-----|--------|-----------|
+|type|`"kinesis"`|Yes|<code>("onetime"&#124;"scheduled"&#124;"sns"&#124;"kinesis")</code> type of scheduling/workflow kick-off desired|
+|value|`<String>`|No|Discussion of valid values is [below](#rule-value)|
+|arn|`<String>`|No|ARN of SNS topic or Kinesis stream (if this is either a SNS Rule or Kinesis Rule)|
```
[the remark this answers is not in the document]
Comment (flamingbear) [reply docs/data-cookbooks/setup.md]: Yes, delete `arn` from this example.  See my previous comments about  value/arn above.
It's set to a uuid, but it's not user configurable I don't think.

File: docs/data-cookbooks/cnm-workflow.md [hunk 3]
```diff
@@ -225,15 +225,21 @@ Cumulus provides a built-in Kinesis consumer lambda function ([kinesis-consumer]
...
+    "name": "${collection_name}", # configured and found in the Collections page
+    "version": "000"              # configured and found in the Collections page
+  },
+  "name": "cnm_basic_rule",
+  "provider": "${provider_id}",   # found on the Provider page
+  "rule": {
+    "type": "kinesis",
+    "value": "arn:aws:kinesis:{{awsRegion}}:{{awsAccountId}}:stream/{{streamName}}",
+    "arn": "arn:aws:kinesis:{{awsRegion}}:{{awsAccountId}}:stream/{{streamName}}"
+  },
```
[the remark this answers is not in the document]
Comment (flamingbear) [reply docs/data-cookbooks/cnm-workflow.md]: I've done this in #500

File: docs/data-cookbooks/setup.md [hunk 4]
```diff
@@ -71,23 +72,36 @@ _The above optional attributes are not shown in the example provided, but they h
...
-|Collection Version|`"006"`|Yes|Version of the collection this rule will moderate. Configured and found in the Collections page|
-|Rule - Type|`onetime`|Yes|<code>(onetime&#124;scheduled&#124;sns&#124;kinesis)</code> type of scheduling/workflow kick-off desired|
-|Rule - Value|[here](#rule-value)|Yes|This entry depends on the type of run|
-|Rule state|`ENABLED`|Yes|<code>(ENABLED&#124;DISABLED)</code> whether or not the rule will be active|
-|Optional tags|`"nightly"`|No|A string type tag that can be added to simplify search|
-
-#### rule-value
+|name|`"L2_HR_PIXC_kinesisRule"`|Yes|Name of the rule. This is the name under which the rule will be listed on the dashboard|
+|workflow|`"KinesisTriggerTest"`|Yes|Name of the workflow to be run. A list of available workflows can be found on the Workflows page|
+|provider|`"PODAAC_SWOT"`|Yes|Configured provider's ID. This can be found on the Providerse page|
```
[the remark this answers is not in the document]
Comment (ifestus) [reply docs/data-cookbooks/setup.md]: Good catch :+1: -- According to our api [schema](https://github.com/nasa/cumulus/blob/master/packages/api/models/schemas.js#L315) `provider` is no longer required, but we still need a `collection`. I'll make this change 😄

File: docs/data-cookbooks/setup.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ifestus) [reply docs/data-cookbooks/setup.md]: Thank you for the explanation 😄. I wasn't sure whether that was a new field or not (and didn't check the api schema). I'll remove this line.

File: docs/data-cookbooks/setup.md [hunk 5]
```diff
@@ -24,14 +24,15 @@ The schema for collections can be found [here](https://github.com/nasa/cumulus/t
...
 |sampleFileName|`"MOD09GQ.A2017025.h21v00.006.2017034065104.hdf"`|Yes|An example filename belonging to this collection|
 |files|`<JSON Object>` of files defined [here](#files)|Yes|Describe the individual files that will exist for each granule in this collection (size, browse, meta, etc.)|
-|provider_path|`"cumulus-test-data/pdrs"`|No|This collection is expecting to find data in a `cumulus-test-data/pdrs` directory, whether that be in S3 or at an http endpoint|
 |dataType|`"MOD09GQ"`|No|Can be specified, but this value will default to the collection_name if not|
 |duplicateHandling|`"replace"`|No|<code>(replace&#124;version&#124;skip)</code> determines granule duplicate handling scheme|
+|process|`"modis"`|No|The options for this are found in "ChooseProcess and in workflows.yml|
+|provider_path|`"cumulus-test-data/pdrs"`|No|This collection is expecting to find data in a `cumulus-test-data/pdrs` directory, whether that be in S3 or at an http endpoint|
+|meta|`<JSON Object>` of MetaData for the collection|No|MetaData for the collection. This metadata will be available to workflowfs for this collection via the Cumulus Message Adapter.
```
[the remark this answers is not in the document]
Comment (ifestus) [reply docs/data-cookbooks/setup.md]: Good question. It looks like meta is still a valid field defined in the api schema for collection.

File: docs/data-cookbooks/setup.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (laurenfrederick) [reply docs/data-cookbooks/setup.md]: @flamingbear Are we saying the rule is not enabled by default? I'm a little confused. For the purpose of this doc, leave state enabled in @ifestus.

File: docs/data-cookbooks/setup.md [hunk 6]
```diff
@@ -74,37 +70,33 @@ Rules are used by to start processing workflows and the transformation process.
...
-|provider|`"PODAAC_SWOT"`|Yes|Configured provider's ID. This can be found on the Providerse page|
-|collection|`<JSON Object>` collection object shown [below](#collection-object)|Yes|Name and version of the collection this rule will moderate. Configured and found in the Collections page|
-|rule|`<JSON Object>` rule type and associated values - discussed [below](#rule-object)
-|state|`ENABLED`|Yes|<code>(ENABLED&#124;DISABLED)</code> whether or not the rule will be active|
+|provider|`"PODAAC_SWOT"`|No|Configured provider's ID. This can be found on the Providers dashboard page|
+|collection|`<JSON Object>` collection object shown [below](#collection-object)|Yes|Name and version of the collection this rule will moderate. Relates to a collection configured and found in the Collections page|
+|rule|`<JSON Object>` rule type and associated values - discussed [below](#rule-object)|Yes|Object defining the type and subsequent attributes of the rule|
+|state|`ENABLED`|No|<code>(ENABLED&#124;DISABLED)</code> whether or not the rule will be active (defaults to `ENABLED`)|
```
Comment (flamingbear) [inline docs/data-cookbooks/setup.md]: so this is still not true yet?  I didn't hear from Lauren if this was going to get fixed or not?

File: docs/data-cookbooks/setup.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (flamingbear) [reply docs/data-cookbooks/setup.md]: Yes, If you don't have a state = enabled in your rule, it won't be active.
https://bugs.earthdata.nasa.gov/browse/CUMULUS-856

**Pull request merged**